app/views/static/getinvolved.html.erb

```diff
@@ -100,11 +101,11 @@
                   <p><strong>Can the application be filled out in languages other than English?</strong></p>
                   <p>Unfortunately, we can only accept applications in English. However, we can provide some level of guidance in Spanish, Portuguese, Bahasa and in select other languages as needed. Please contact gfwfund@wri.org for more information.</p>
                   <p><strong>My project seeks funding to purchase equipment, seedlings, or other materials. Can I use the SGF award for this purpose?</strong></p>
-                  <p>No. Given the short term of projects, we cannot award funding for this purpose. However, a portion of funding requested may go towards equipment, such as portable internet sticks, but we typically request that this amount be below $5,000 USD.</p>
+                  <p>No. Given the short term of projects, we cannot award funding for this purpose. However, a portion of funding requested may go towards equipment, such as portable Internet sticks, but we typically request that this amount be below US$5,000.</p>
                   <p><strong>I applied to the SGF last year, but was not accepted. Will my application be given special consideration?</strong></p>
                   <p>No. All applications will be reviewed against the same criteria. However, we encourage previous applicants to apply.</p>
                   <p><strong>I’m not sure what Global Forest Watch is. How can I determine whether this platform can be used for my project?</strong></p>
-                  <p>All applications must describe how GFW will play a critical role in the project. To find out more information about GFW, visit the about or how to pages. You can also view a summary of previous projects in the <a href="https://gfw.fluidreview.com/pm/resource/eyJoZnJlIjogYWh5eSwgInZxIjogODk1M30/" target="_blank">Guidelines for Applicants</a>.</p>
+                  <p>All applications must describe how GFW will play a critical role in the project. To find out more information about GFW, visit the <a href="http://gfw-nav.herokuapp.com/about" target="_blank">About</a> or <a href="http://gfw-nav.herokuapp.com/howto" target="_blank">How to</a> pages. You can also view a summary of previous projects in the <a href="https://gfw.fluidreview.com/pm/resource/eyJoZnJlIjogYWh5eSwgInZxIjogODk1M30/" target="_blank">Guidelines for Applicants</a>.</p>
```

Comment:
Hi! As these are internal links, would you mind removing the 'http://gfw-nav.herokuapp.com' so they'll work on PRO once the project is deployed? Thank you!!